base/statistics.jl

```diff
@@ -512,49 +512,86 @@ median{T}(v::AbstractArray{T}, region) = mapslices(median!, v, region)
 # see ?quantile in R -- this is type 7
 # TODO: need faster implementation (use select!?)
 #
-function quantile!(v::AbstractVector, q::AbstractVector)
-    isempty(v) && throw(ArgumentError("empty data array"))
-    isempty(q) && throw(ArgumentError("empty quantile array"))
+"""
+    quantile!([q, ] v, p; sorted=false)
+
+Compute the quantile(s) of a vector `v` at the probabilities `p`, with optional output into array `q` (if not provided, a new output array is created). The keyword argument `sorted` indicates whether `v` can be assumed to be sorted; if `false` (the default), then the elements of `v` may be partially sorted.
```

Comment:
line breaks

and should be sure the rst sig is consistent, and rerun genstdlib